script.js

```diff
@@ -516,21 +518,12 @@ function handleCountiesLayer(geojson) {
   const layerOptions = {
     style: feature => {
       // style counties based on whether their moratorium has passed
-      if (feature.properties.passed === "Yes") {
-        return {
-          color: "#4dac26",
-          fillColor: "#b8e186",
-          fillOpacity: fillOpacity,
-          weight: strokeWeight
-        };
-      } else {
-        return {
-          color: "#d01c8b",
-          fillColor: "#f1b6da",
-          fillOpacity: fillOpacity,
-          weight: strokeWeight
-        };
-      }
+      return {
+        color: strokeColorScale[feature.properties.range] || colorNoData,
```

Comment:
If we are using this boolean to return the no data colour, then why set the 0 index of the list to be also `colorNoData`?

Reply:
Agree, we could set the 0 index to undefined